Assets/MRTK/Core/Interfaces/Services/IMixedRealitySystem.cs

```diff
@@ -6,9 +6,9 @@
 namespace Microsoft.MixedReality.Toolkit
 {
     /// <summary>
-    /// Generic interface for all Mixed Reality Services
+    /// Base interface for all Mixed Reality systems
     /// </summary>
-    public interface IMixedRealityService : IDisposable
+    public interface IMixedRealitySystem : IDisposable
```

Comment:
Do we need to update all the derived classes too?

Reply:
i'll look into that